packages/omgidl-parser/src/processIDL.ts

```diff
@@ -44,6 +45,19 @@ export function buildMap(definitions: AnyASTNode[]): Map<string, AnyIDLNode> {
   return idlMap;
 }
 
+function getValueAnnotation(
+  annotations: Record<string, AnyAnnotation> | undefined,
+): UnresolvedConstantValue | ConstantValue | undefined {
+  if (!annotations) {
+    return undefined;
+  }
+  const valueAnnotation = annotations["value"];
+  if (valueAnnotation && valueAnnotation.type === "const-param") {
```

Comment:
Is there some long form syntax like `@value(something=1)` that we should also support?

Reply:
Not that I can see anywhere. Everywhere its used in the DDS spec it's used with just a single constant parameter